modules/tiled-gallery/tiled-gallery/tiled-gallery-layout.php

```diff
@@ -42,7 +42,16 @@ private function template( $name, $context = null ) {
 			extract( $context );
 		}
 
-		require dirname( __FILE__ ) . "/templates/$name.php";
+		/**
+		 * Filters the Tiled Gallery template path
+		 *
+		 * @since 4.2.3
```

Comment:
Could you update this to use `@since 4.4.0` instead?

I would also recommend adding `@module tiled-gallery` above the `@since` declaration.

Thanks!